ietf-optical-impairment-topology.tree

```diff
@@ -28,15 +29,21 @@ module: ietf-optical-impairment-topology
                 |  +--ro amplifier
                 |     +--ro type-variety    string
                 |     +--ro operational
-                |        +--ro actual-gain                             decimal64
-                |        +--ro tilt-target                             decimal64
-                |        +--ro out-voa                                 decimal64
-                |        +--ro in-voa                                  decimal64
+                |        +--ro actual-gain
+                |        |       decimal64
+                |        +--ro tilt-target
+                |        |       decimal64
+                |        +--ro out-voa
+                |        |       decimal64
+                |        +--ro in-voa
+                |        |       decimal64
                 |        +--ro (power-param)?
                 |           +--:(channel-power)
-                |           |  +--ro nominal-channel-power?            decimal64
+                |           |  +--ro nominal-channel-power?
+                |           |          decimal64
```

Comment:
do we have a crisp definition of "channel"? if it is of variable width, what is then the "channel power" per AMP if each channel has a different width?

Reply:
@ggrammel : Thanks Gert, I've open a new issue for your comment.